### ISSUE TITLE
Use method group instead of extension method

### ISSUE THREAD

**Issue by jnyrup** [JosefPihrt/Roslynator#173]
I believe RCS1207 could be extended to also cover extension methods

```c#
public static Target Convert(this Source) => new Target();

var targets = sources.Select(s => Convert(s)); // Triggers RCS1207

var targets = sources.Select(s => s.Convert()); // Does not trigger
```
